10up/ElasticPress
# Fix/2641

### Description of the Change

Correct the link on the Index Health page when no indexes are found.


Closes #2641.

### Verification Process

1. Delete any indexes via WP CLI, `wp elasticpress delete-index`.
2. Visit _ElasticPress > Index Health_.
3. You'll see a message, "We could not find any data for your Elasticsearch indices. Maybe you need to sync your content?". Follow the "sync your content" link.
4. You should be taken to the _ElasticPress > Sync_ page.

### Checklist:
